.tool-versions

```diff
@@ -1,2 +1,2 @@
 scarb 2.6.3
-starknet-foundry 0.20.0
+starknet-foundry 0.21.0
```

Comment:
Could you update the Dockerfiles to reflect this change, like here https://github.com/keep-starknet-strange/art-peace/blob/bf05739873151c4bd9fa457f05d21f317a5401da/backend/Dockerfile#L12

Either manually change them to `0.21.0` or set up a way for the Dockerfile to properly import the changed value if you'd find that interesting.

Reply:
@b-j-roberts  will change manually for now if this is ok for you